app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -187,6 +200,32 @@ export const ContentForm: React.FC<IContentFormProps> = ({ contentType }) => {
     }
   };
 
+  const handleTranscribe = async (values: IContentForm) => {
```

Comment:
Moved `Request Transcription` and `Request NLP` to the bottom of the form instead of on the Transcription tab.